javascripts/discourse/initializers/initialize-for-easy-footer.js

```diff
@@ -5,6 +5,7 @@ export default {
   name: "show-footer-on-static-pages",
   initialize() {
     withPluginApi("0.8", api => {
+      pluginId: "Discourse-easy-footer",
       if (settings.Show_footer_on_login_required_page) {
         api.modifyClass("controller:static", {
```

Comment:
```suggestion
      if (settings.Show_footer_on_login_required_page) {
        api.modifyClass("controller:static", {
          pluginId: "Discourse-easy-footer",
```

Reply:
Thanks! Sorry about that. :-(

Reply:
Oh! Funny. There is! How did you know?!?!

Thanks very much. I'm incredibly bad at this.